src/main/webapp/projects.html

```diff
@@ -27,7 +27,7 @@
     <!-- Note: when deploying, replace "prop-types.js" with "prop-types.min.js" -->
     <script src="https://unpkg.com/prop-types@15.6/prop-types.js"></script>
     <!-- Load React components -->
-    <script type="module" src="./js/common/react/jsx-processed/Navbar.js"></script>
+    <script type="module" src="./js/common/react/jsx-processed/display_navbar.js"></script>
```

Comment:
Is there documentation for importing the navbar? It might be good to have a comment in the file explaining what link to add.

Reply:
That would make sense. Will add some basic instructions in the `display_navbar.js` file.